Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+  </data>
+  <data name="PackageViewModel_DisplayName" xml:space="preserve">
+    <value>Pakke - {0}</value>
+  </data>
+  <data name="PackageViewModel_FailedToInstall" xml:space="preserve">
+    <value>Mislyktes i å Installere</value>
+  </data>
+  <data name="PackageViewModel_FailedToPin" xml:space="preserve">
+    <value>Mislyktes i å Feste</value>
+  </data>
+  <data name="PackageViewModel_FailedToReinstall" xml:space="preserve">
+    <value>Mislyktes i å Reinstallere</value>
+  </data>
+  <data name="PackageViewModel_FailedToUninstall" xml:space="preserve">
+    <value>Mislyktes i å Avinstallere</value>
+  </data>
+  <data name="PackageViewModel_FailedToUnpin" xml:space="preserve">
+    <value>Mislyktes i å Løsne</value>
+  </data>
+  <data name="PackageViewModel_FailedToUpdate" xml:space="preserve">
+    <value>Mislyktes i å Oppdatere</value>
+  </data>
+  <data name="PackageViewModel_InstallingPackage" xml:space="preserve">
+    <value>Installerer pakke</value>
+  </data>
+  <data name="PackageViewModel_LoadingPackageInfo" xml:space="preserve">
+    <value>Laster inn pakkeinformasjon...</value>
+  </data>
+  <data name="PackageViewModel_PinningPackage" xml:space="preserve">
+    <value>Fester pakke</value>
+  </data>
+  <data name="PackageViewModel_RanIntoInstallError" xml:space="preserve">
+    <value>Det oppsto en feil under installeringen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoPinningError" xml:space="preserve">
+    <value>Det oppsto en feil under festingen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoReinstallError" xml:space="preserve">
+    <value>Det oppsto en feil under reinstalleringen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoUninstallError" xml:space="preserve">
+    <value>Det oppsto en feil under avinstalleringen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoUnpinError" xml:space="preserve">
+    <value>Det oppsto en feil under løsningen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoUpdateError" xml:space="preserve">
+    <value>Det oppsto en feil under oppdateringen av {0}.
+{1}</value>
+  </data>
+  <data name="PackageViewModel_ReinstallingPackage" xml:space="preserve">
+    <value>Reinstallerer pakke</value>
+  </data>
+  <data name="PackageViewModel_UninstallingPackage" xml:space="preserve">
+    <value>Avinstallerer pakke</value>
+  </data>
+  <data name="PackageViewModel_UnpinningPackage" xml:space="preserve">
+    <value>Løsner pakke</value>
+  </data>
+  <data name="PackageViewModel_UpdatingPackage" xml:space="preserve">
+    <value>Oppdaterer pakke</value>
+  </data>
+  <data name="RemoteSourceViewModel_FailedToLoad" xml:space="preserve">
+    <value>Kunne ikke laste inn</value>
+  </data>
+  <data name="RemoteSourceViewModel_FailedToLoadRemotePackages" xml:space="preserve">
+    <value>Kunne ikke laste inn eksterne pakker!

```

Comment:
Merkelig.

@gep13: Why is question mark used to terminate this error message and a few others, but not all?